js/app.js

```diff
@@ -0,0 +1,367 @@
+		    "wool scarf",
+		    "raybans"
+		  ]
+		];
+		//1.
+		console.log("Kristyn is rocking that " + kristynsCloset[2] + " today!");
+		//2.
+		kristynsCloset.splice(6,0,"raybans");
+		//3. 
+		kristynsCloset[5] = "stained knit hat";
+		//4.
+		console.log(thomsCloset[0][0]);
+		//5.
+		console.log(thomsCloset[1][1]);
+		//6.
+		console.log(thomsCloset[2][2]);
+		//7.
+		console.log("Thom is looking fierce in a " + thomsCloset[0][0] + ", " + thomsCloset[1][1] + " and " + thomsCloset[2][2] + "!");
+		//8.
+		thomsCloset[1][2] = "Footie Pajamas";
+//IV. Functions
+	//A. printGreeting
+	//skipped
+	//B. printCool
+	const printCool = (name) => {
+		return name + " is cool";
+	}
+	console.log(printCool("Jeff"));
+	//C. calculateCube
+	const calculateCube = (number) => {
+		return number**3;
+	}
+	console.log(calculateCube(4));
+	//D. isVowel
+	const isVowel = (char) => {
+		if(char === "a" || char === "A" || char === "e" || char === "E" || char === "i" || char === "I" || char === "o" || char === "O" || char === "u" || char === "U") {
+			return true;
+		} else {
+			return false;
+		}
+	}
+	//E. getTwoLengths
+	const getTwoLengths = (str1,str2) => {
+		return [str1.length,str2.length];
+	}
+	//F. getMultipleLengths
+	const getMultipleLengths = (strArray) => {
+		let numArray = [];
+		for(let i = 0; i < strArray.length; i++) {
+			numArray.push(strArray[i].length);
+		}
+		return numArray;
+	}
+	//G. maxOfThree
+	const maxOfThree = (num1,num2,num3) => {
+		if(num1 >= num2 && num1 >= num3) {
+			return num1;
+		} else if(num2 >= num1 && num2 >= num3) {
+			return num2;
+		} else {
+			return num3;
+		}
+	}
+	//h. printLongestWord
+	const printLongestWord = (array) => {
+		array.sort(function(a,b) {
+			return b.length - a.length;
+		})
+		return array[0];
+	}
+	//i. transmogrify
+	const transmogrify = (num1,num2,num3) => {
+		return (num1*num2)**num3;
+	}
+	//j. reverseWordOrder v2
+	
+
+	const reverseWordOrder = (string) => {
+		let newString = "";
+		let currentWord = "";
+		for(let i = 0; i < string.length; i++) {
+			let prevChar = "";
+			//console.log(prevChar);
+			if(string[i] !== " ") {
+				currentWord = currentWord + string[i];
+				//console.log(currentWord);
+				//console.log(newString);
+			} else if(string[i] === " ") {
+				newString = currentWord + string[i] + newString;
+				currentWord = "";
+				//console.log(currentWord);
+				//console.log(newString);
+			}
+			prevChar = string[i];
+			//console.log(prevChar);
+			//console.log(newString);
+		}
+		newString = currentWord + " " + newString;
+		console.log(newString);
+	}
+	//"food like I"
+	//k. Math.random()
+	//1. int between 1 and 10
+	/*const rand1 = () => {
+		return Math.floor(Math.random()*9)+1;
+	}
+	//2. int between 10 and 100
+	const rand2 = () => {
+		return Math.floor(Math.random()*90)+10;
+	}
+	//3. between 532 and 13267
+	const rand3 = () => {
+		return Math.random()*12735+532;
+	}
+	//4. between 1 and 10
+	const rand4 = () => {
+		return Math.random()*9+1;
+	}
```

Comment:
same as previous, i'll take it since it said "number" (good job noticing that) but not without pointing out it can't give a value higher than 9.99999999999..